### PR TITLE
Replace static method with an object

### PR DESCRIPTION
**Pull request by victornoel**
This is for #18

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#32](https://github.com/g4s8/ghman/issues/32) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#32](https://github.com/g4s8/ghman/issues/32) is assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CJT4R32Q3/ec0fa106-2b97-4b87-8b3e-6afa0829219e); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/main/java/com/g4s8/ghman/data/PgUsers.java [hunk 1]
```diff
@@ -99,24 +100,37 @@ public User user(final long uid) {
...
+     * User details as json from telegram chat.
+     *
+     * @since 1.0
      */
-    private static JsonObject details(final Chat chat) {
-        final JsonObjectBuilder details = Json.createObjectBuilder()
-            .add("firstName", chat.getFirstName());
-        if (chat.getLastName() != null) {
-            details.add("lastName", chat.getLastName());
+    private static final class JsonUserDetails implements Scalar<JsonObject> {
```
Comment (olenagerasimova) [inline src/main/java/com/g4s8/ghman/data/PgUsers.java]: @victornoel please, implement unit test for this class

**Comment by olenagerasimova** [comment]
@victornoel ping

**Comment by 0crat** [comment]
@olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by victornoel** [comment]
@olenagerasimova sorry, I will do it soon

**Comment by victornoel** [comment]
@olenagerasimova thanks for the wait, I've added instruction to continue the work and add tests at one point

**Comment by olenagerasimova** [comment]
@victornoel you have to decrease code coverage, build failed. And please next time try to implement new functionality and unit test for it together. I am not comfortable approving PR having no idea whether new peace of code works or not. 
@g4s8 when you will merge this PR please take a look at todo to continue this job: it seems to me that extracting interface in this case is an overhead.

**Comment by olenagerasimova** [comment]
@victornoel ping

**Comment by victornoel** [comment]
@olenagerasimova thx, I missed the failing build. Btw don't worry about not implementing tests yet for this, it's just a private inner class for now, it is tested as part of the tests of the outer class, when it gains more behaviour then it will become important I believe. Also it's the idea of PDD to do things by small chunks ;)

**Comment by olenagerasimova** [comment]
@victornoel and are you going to make build pass?

**Comment by victornoel** [comment]
@olenagerasimova I thought I did, it should be alright now, thx for the heads up

**Comment by olenagerasimova** [comment]
@victornoel thanks

**Comment by olenagerasimova** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Code review was too long (9 days), architects (@g4s8) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
The job [#32](https://github.com/g4s8/ghman/issues/32) is now out of scope

**Comment by 0crat** [comment]
@ammaratef45[/z](https://www.0crat.com/u/ammaratef45) please review this job completed by @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Job [#32](https://github.com/g4s8/ghman/issues/32) is not in the agenda of @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), can't inspect

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ammaratef45** [comment]
@olenagerasimova 
You've found less than three problems in the code, please confirm you'll try to find at least three problems in the code next time

**Comment by olenagerasimova** [comment]
@ammaratef45 I confirm

**Comment by ammaratef45** [comment]
@olenagerasimova Thanks

**Comment by ammaratef45** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45)

**Pull request merged**